js/app.js

```diff
@@ -0,0 +1,457 @@
+  {zodiac: 'Horse', enemy: "Rat", tag:"a7", url: "./assets/designs_horse.png", desc: 'Horse is a free-spirited animal.', luck: 'Luck: -2'},
+  {zodiac: 'Goat', enemy: "Ox", tag:"a8", url: "./assets/designs_goat.png", desc: 'Goat is a peaceful animal.', luck: 'Luck: -10'},
+  {zodiac: 'Monkey', enemy: "Tiger", tag:"a9", url: "./assets/designs_monkey.png", desc: 'Monkey is a playful animal.', luck: 'Luck: -100'},
+  {zodiac: 'Rooster', enemy: "Rabbit", tag:"a10", url: "./assets/designs_rooster.png", desc: 'Rooster is a worried animal.', luck: 'Luck: +20'},
+  {zodiac: 'Dog', enemy: "Dragon", tag:"a11", url: "./assets/designs_dog.png", desc: 'Dog is a friendly animal.', luck: 'Luck: +35'},
+  {zodiac: 'Pig', enemy: "Pig", tag:"a12", url: "./assets/designs_pig.png", desc: 'Pig is a hungry animal.', luck: 'Luck: +40'}
+]
+
+/*---------------------------- Variables (state & etc) ----------------------------*/
+let slotMachineArray, scoresArray, turn, round, isWinner, sumA, sumB, playerAName, playerBName, count, numberSlot4
+
+/*------------------------ Cached Element References ------------------------*/
+const choosePlayer = document.querySelector('#choose-player')
+const slotMachine = document.querySelector('#slot-machine')
+const scoreBoard = document.querySelector('#score-board')
+
+const favicon = document.querySelector('#favicon')
+const tooltips = document.querySelectorAll('[data-toggle="tooltip"]')
+tooltips.forEach(tooltip => tooltip.tooltip())
+
+/*----------------------------- Event Listeners -----------------------------*/
+slotMachine.querySelector('#replay-button').addEventListener("click", init)
+choosePlayer.querySelector('#scroll').addEventListener("click", showPlayer)
+choosePlayer.querySelector('#select-button').addEventListener("click", selectPlayer)
+choosePlayer.querySelector('#play-button').addEventListener("click", handlePlay)
+slotMachine.querySelector('#shuffle-button').addEventListener("click", handleShuffle)
+
+
+
+/*-------------------------------- Functions --------------------------------*/
+
+init()
+
+function init() {
+  slotMachineArray = [slot1, slot2, slot3, slot4]
+  slotMachineArray.forEach(slot => {
+    slot.className = "a3"
+  })
+  
+  turn = 1
+  isWinner = null
+  round = 0
+  sumA = 0
+  sumB = 0
+  playerAName = ''
+  playerBName = ''
+  
+  scoresArray = [score0, score1, score2, score3, score4, score5, score6, score7]
+  score0.target = {player: 1, round: 0, score: 0, note: ''}
+  score1.target = {player: -1, round: 1, score: 0, note: ''}
+  score2.target = {player: 1, round: 2, score: 0, note: ''}
+  score3.target = {player: -1, round: 3, score: 0, note: 'sum'}
+  score4.target = {player: 1, round: 4, score: 0, note: ''}
+  score5.target = {player: -1, round: 5, score: 0, note: ''}
+  score6.target = {player: 1, round: 6, score: 0, note: ''}
+  score7.target = {player: -1, round: 7, score: 0, note: 'sum'}
```

Comment:
This is a really cool idea for you to store you state, but you could optimize it a bit. For example:

```javascript
  score0.target = {player: 1, round: 0, score: 0, note: ''}
```

could become

```javascript
const score0 = {player: 1, round: 0, score: 0, note: ''}
```

then fill the array with the scores, just like you're doing on line 58 now. That means you don't need to use `.target` anymore (`.target` isn't important outside of the context of an event)